Tests/ArgumentParserUnitTests/UsageGenerationTests.swift

```diff
@@ -15,6 +15,17 @@ import XCTest
 final class UsageGenerationTests: XCTestCase {
 }
 
+func _testSynopsis<T: ParsableArguments>(
+  _ type: T.Type,
+  visibility: ArgumentVisibility = .default,
+  expected: String,
+  file: StaticString = #file,
+  line: UInt = #line
+) {
+  let help = UsageGenerator(toolName: "example", parsable: T(), visibility: visibility)
+  XCTAssertEqual(help.synopsis(), expected, file: file, line: line)
```

Comment:
`visibility` doesn't get passed to `help.synopsis` here. (this is why I don't like the default value)